packages/connect/src/utils/pathUtils.ts

```diff
@@ -1,5 +1,7 @@
 // origin: https://github.com/trezor/connect/blob/develop/src/js/utils/pathUtils.js
 
+import { cloneObject } from '@trezor/utils';
```

Comment:
so it can go away now after https://github.com/trezor/trezor-suite/pull/14959/files#diff-8c98ba84fc78374812ebc5a974004cbaf72f6245466b70f82c6e0a53f8eadcd3R210 was added?

Reply:
We probably could, but why do you think it will be better?
For me, an utility function that modifies the passed attribute, sounds like a bad practice.

Reply:
please remove it 🙏 I would like to validate that the new global utility really does the job

Reply:
https://github.com/trezor/trezor-suite/pull/14959/commits/a30a0a676b248ac173a0b525d07ad8704efb916c